frontend/website/mkdocs.yml

```diff
@@ -22,6 +22,12 @@ nav:
     - Lifecycle of a Payment: architecture/lifecycle-payment.md
     - Consensus: architecture/consensus.md
     - Proof of Stake: architecture/proof-of-stake.md
+  - SNARKs:
```

Comment:
I think this should go under the developers umbrella, not be top level — also if this is something that we’re not going to maintain carefully and just use within workshops, we should consider not adding it to the index at all (this is what we’re doing with the hackathon doc page we made)

Reply:
Can you have nested things under developers? I couldn't get it to work.

Reply:
this issue says its doable: https://github.com/mkdocs/mkdocs/issues/1139

but I'm not able to get this to work either:

```
  - Developers:
    - Developers Overview: developers/index.md
    - Codebase Overview: developers/codebase-overview.md
    - Repository Structure: developers/directory-structure.md
    - Code Reviews: developers/code-reviews.md
    - Style Guide: developers/style-guide.md
    - GraphQL API: developers/graphql-api.md
    - SNARKs:
      - SNARKs Overview: developers/snarks/index.md
      - Getting started using SNARKs: developers/snarks/snarky.md
      - Which SNARK is right for me?: developers/snarks/constructions.md
      - The snarkyjs-crypto library: developers/snarks/snarkyjs-crypto.md
      - The snarky-universe library: developers/snarks/snarky-universe.md
```